release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -2,27 +2,14 @@
 layout: ballerina-blank-page
 title: Release Note
 ---
-### Overview of Ballerina Swan Lake Alpha3
+<em>This is the third alpha in a series of planned alphas and betas leading up to the Ballerina Swan Lake GA release.</em> 
 
-This Alpha3 release includes the language features planned for the Ballerina Swan Lake release. Moreover, this release includes improvements and bug fixes to the compiler, runtime, standard library, and developer tooling. This release note lists only the features and updates added after the Alpha2 release of Ballerina Swan Lake.
+Ballerina Swan Lake Alpha3 contains new language features, improvements, and bug fixes to the compiler, runtime, standard library, and developer tooling. 
```

Comment:
```suggestion
Ballerina Swan Lake Alpha3 contains new language features, improvements, and bug fixes to the compiler, runtime, standard library, and developer tooling. 
```
We specifically wanted to say that these alpha releases include the features that are planned for the GA release.

@MaryamZi - WDYT?

Reply:
Yeah, it would be better if we can continue to say that this release includes the features planned for GA. But I think we can improve the current version too. How about something along the lines of

```suggestion
Ballerina Swan Lake Alpha3 introduces new language features planned for the Ballerina Swan Lake GA release. Moreover, this release includes improvements and bug fixes to the compiler, runtime, standard library, and developer tooling.
```

Reply:
introduces vs includes because I think what we want to say here is that new features have been introduced between alpha2 and alpha3.

Reply:
+1. This was changed if we were going to mention that snippet at the beginning about alpha3 leading up to the Swan Lake GA release. BTW, what other components does Ballerina mainly consist of other than compiler, runtime, standard library and tooling? Something that's not mentioned here or didn't include any bug fixes or improvements?

Reply:
Changed as follows:

"This Ballerina Swan Lake <VERSION> introduces new language features planned for the Swan Lake GA release. Moreover, this release includes improvements and bug fixes done to the compiler, runtime, standard library, and developer tooling after the after the Swan Lake <PREVIOUS_VERSION> release."